Mathlib/MeasureTheory/Measure/LevyProkhorovMetric.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2023 Kalle Kytölä. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kalle Kytölä
 -/
-import Mathlib.MeasureTheory.Measure.ProbabilityMeasure
+import Mathlib.MeasureTheory.Measure.Portmanteau
```

Comment:
This strengthened import is necessary for the nontrivial direction of the comparison of the Lévy-Prokhorov topology and the weak convergence topology. So it was just a matter of time and should not be seen as import creep".